content/en/tracing/profiler/enabling/linux.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Enabling the Datadog Profiler for Linux
+kind: Documentation
+code_lang: linux
+type: multi-code-lang
+code_lang_weight: 90
+further_reading:
+    - link: 'getting_started/profiler'
+      tag: 'Documentation'
+      text: 'Getting Started with Profiler'
+    - link: 'tracing/profiler/profiler_troubleshooting'
+      tag: 'Documentation'
+      text: 'Fix problems you encounter while using the profiler'
+---
+
+{{< site-region region="us5" >}}
+<div class="alert alert-warning">
+  The Continuous Profiler is not available for the Datadog {{< region-param key="dd_site_name" >}} site.
+</div>
+{{< /site-region >}}
+
+<div class="alert alert-warning">
+The Datadog Profiler for Linux is in public beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
+</div>
+
+The Datadog Profiler for Linux lets you collect profile data for applications running on Linux.  It is ideally suited for applications written in compiled languages, such as C, C++, or Rust.  It can also profile services such as nginx, Redis, or Postgres, as it does not require access to source code.
+
+## Requirements
+
+- The Datadog Profiler requires Linux kernel v4.17+ on an `amd64` compatible processor. It does not support macOS, BSD, Windows, or other operating systems besides Linux v4.17 or later. 
+- The `/proc/sys/kernel/perf_event_paranoid` setting must *at most* 2, or `CAP_SYS_ADMIN` (or `CAP_PERFMON` on Linux v5.8 or later) must be granted to the profiling process or service container.
+
+## Installation
+
+To begin profiling applications:
+
+1. Ensure you are running the Datadog agent version [7.20.2][1]+ or [6.20.2][2]+ and that you satisfy the profiler requirements.
+
+2. Download the appropriate [ddprof binary][3] for your Linux distribution.  For example, here is one way to pull the latest release:
+
+```shell
+curl -L -o- https://github.com/DataDog/ddprof/releases/download/v0.7.0/ddprof-x86_64_unknown-linux-gnu-2.23.tar.gz | tar xz --strip-components 2 ddprof-x86_64_unknown-linux-gnu-2.23/bin/ddprof
```

Comment:
@r1viollet @dbenamydd @kayayarai, I think this example adds some clarification for users.  It has the following advantages
 * one-liner
 * doesn't presume any particular shell.
 * retrieves ddprof into the current directory, rather than in a deeply nested folder (which we sort of need to have, since we are obligated to ship licenses alongside artifacts)
 * doesn't leave behind an annoying temporary tar.gz
 
 The problem is, this line is long enough that it breaks in an awkward place when represented on the page.  Is this unbearably annoying for anyone else?
 
 Ideas:
 * split it up into multiple lines (not ideal:  annoying when a user has to copy-paste a multiline command)
 * package the licenses independently from the release (I think we still satisfy our open source requirements if we do this, right?)
 * don't even tar ddprof in the first place (not ideal:  ddprof compresses to 30% of its final size, but then again this is only the difference between 5 and 15 megabytes, so no big deal)
 
 Any thoughts?

Reply:
Oh hah I just did something similar but didn't push yet. Subjective 2 cents: a multiline command isn't a problem. Eg my version was
 ```shell
 wget -O ddprof.tar.gz https://github.com/DataDog/ddprof/releases/download/v0.7.0/ddprof-x86_64_unknown-linux-gnu-2.23.tar.gz
 tar xf ddprof.tar.gz
 mv ddprof/bin/ddprof INSTALLATION_TARGET
 ```

Reply:
OK, added something like that.  Thanks!